core-war/src/main/java/org/silverpeas/web/pdc/control/PdcSearchSessionController.java

```diff
@@ -1061,8 +871,8 @@ private void setExtraInfoToResultsToDisplay(List<GlobalSilverResult> results) {
                 componentId);
             if (componentInst != null) {
               String title = componentInst.getLabel(getLanguage());
-              result.setTitle(title);
-              result.setType("Wysiwyg");
+              result.setName(title);
+              //result.setType("Wysiwyg");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3ACommentedOutCodeLine)